lib/portus/jwt_token.rb

```diff
@@ -8,10 +8,10 @@ class JwtToken
     # The constructor takes the query parameters as specified in the
     # specification. The given scope is assumed to have already been processed
```

Comment:
Could you update `The given scope is assumed to have already been processed` to  `The given scopes are assumed to have already been processed` ?